src/js/view/reducers/logViewerReducer.js

```diff
@@ -118,6 +119,19 @@ export const logViewerReducer = (state = initialState, action) => {
       };
     }
 
+    //updateScrollPosition action passed by Reducer
```

Comment:
This comment is probably not needed here. Or what do you think? I find the action.type explaining what is going on in itself.

Reply:
Yes right that's not needed, so i just removed it.